Dolphin scripts/README.md

```diff
@@ -16,13 +27,26 @@
 6. In Dolphin, under "Scripts", click "Add new Scripts" and select `Scripts/Entrance Randomizer/__main__.py`.
 7. Enjoy and watch logs for errors 🙂
 
+### About the .graphml file
+
+In order to display the generated map take these steps:
+1. Go to https://graphonline.ru/en/
+2. Under "Graph" choose "Import from file" and choose your generated .graphml file
+3. Under "Algorithms" choose "Arrange the graph"
+4. Enjoy your map! You can freely move the dots, and you can use multiple features under the "Algorithms" tab (like "Find all paths" for instance)
+5. You can also save your map at any time by choosing "Graph" --> "Export to file"
+
 ### Known issues and limitations
 
-- To generate a new seed, simply reload the script.
-- Some linked transitions are not spawning at the right entrance and use the default entrance instead. Known cases:
-  - Jungle Canyon from Punchau Shrine
-  - Bittenbinder's Camp from Mysterious Temple
-- One-way transitions are not linked together. This can be a bit confusing when sometimes going back leads to a different area.
+- To generate a new seed, simply reload the script
+- Sometimes a transition does not make you enter the level from the correct entrance, but from the default entrance instead
```

Comment:
I'd like to keep the known case, they're also a reminder of those I found so far. But maybe add a sentence indicating that there's probably a lot more and it has to be investigated?

Reply:
Except that your 'known case' list isn't very accurate. I've witnessed many, many transitions that sometimes work perfectly, and other times use the default entrance instead. It's a bug that more or less randomly appears during runtime and that we'll need to fix at some point.

I think the current "Known issues and limitations: Sometimes a transition does not make you enter the level from the correct entrance, but from the default entrance instead" sums up the situation pretty well. If you don't agree with this phrasing feel free to suggest an alternative.

Reply:
If the same transition works randomly, that's a different bug.
I'm talking about levels consistently not using "previous area id", or using a completely different id tan you'd expect (like Statues to BBCamp)

can you keep the two I dentified in a comment?
```md
<!--
stuff here won't render
-->
```

Reply:
My list of levels that consistenly don't use "previous area id" is currently 0. All levels that don't use it sometimes either do use it some other times, or I haven't tested it thoroughly enough.
Once I fully test levels this way and find levels to put in that list I'll add them in the Readme, until then I'll mark this as resolved.